web_src/js/features/admin/common.js

```diff
@@ -50,11 +50,11 @@ export function initAdminCommon() {
 
   function onUsePagedSearchChange() {
     if ($('#use_paged_search').prop('checked')) {
```

Comment:
The following is split into different parts because GitHubs selection algorithm is buggy.
```suggestion
    const $pageSize = $('.search-page-size');
    if ($('#use_paged_search').prop('checked')) {
```

Reply:
I really do not want to touch the old code too much, since this PR is only a bug fix.

TBH, the `attr('required', 'required');` is also incorrect. If I would like to improve, I would have rewritten them all.